Parsers/ASimAuditEvent/Parsers/ASimAuditEventVMwareCarbonBlackCloud.yaml

```diff
@@ -0,0 +1,186 @@
+Parser:
+  Title: Audit Event ASIM parser for VMware Carbon Black Cloud
+  Version: '0.1.0'
+  LastUpdated: Oct 23 2023
+Product:
+  Name: VMware Carbon Black Cloud
+Normalization:
+  Schema: AuditEvent
+  Version: '0.1'
+References:
+- Title: ASIM Audit Event Schema
+  Link: https://aka.ms/ASimAuditEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: VMware Carbon Black Cloud documentation
+  Link: 
+    https://developer.carbonblack.com/reference/carbon-black-cloud/data-forwarder/schema/latest/endpoint.event-1.0.0/
+    https://developer.carbonblack.com/reference/cb-threathunter/latest/event-search-fields/
+Description: |
+  This ASIM parser supports normalizing VMware Carbon Black Cloud logs to the ASIM Audit Event normalized schema. VMware Carbon Black Cloud events are captured through VMware Carbon Black Cloud data connector which ingests Carbon Black Audit, Notification and Event data into Microsoft Sentinel through the REST API.
+ParserName: ASimAuditEventVMwareCarbonBlackCloud
+EquivalentBuiltInParser: _ASim_AuditEvent_VMwareCarbonBlackCloud
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+    let EventTypeLookup = datatable(temp_type: string, EventType: string)[
+        "created", "Create",
+        "updated", "Set",
+        "deleted", "Delete",
+        "added", "Create",
+        "modified", "Set"
+    ];
+    let parser = (disabled: bool=false) {
+        let allData = CarbonBlackAuditLogs_CL
+            | where not(disabled)
+            | where not(description_s has_any ("logged in", "login"));
```

Comment:
@jayeshprajapaticrest @niralishah-crest 
other potential audit events I found:
Sent deregister request to 1 device(s): xxxxxx
Re-registration of device xxxx\xxxxx of xxxxx device completed
xxx\xxxx deregistered through auto de-registration of CWP workloads
Successfully deleted deregistered device(s): xxxxx

Reply:
@jayeshprajapaticrest @niralishah-crest don't worry about this. Will take care of these.